scripts/bazel/build-packages.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Exit if any of the builds fails
+set -e
+
+# Source directories for packages to build with bazel and copy into dist/
+packages=(cdk lib cdk-experimental material-experimental material-moment-adapter)
+
+# The bazel-bin directory where the bazel output is written
+bazel_bin="$(bazel info bazel-bin)"
+
+# The dist/ directory where we want to copy the final result. Clear it out to avoid
+# any artifacts from earlier builds being retained.
+packages_dist="./dist/bazel-packages"
+rm -rf ${packages_dist}
+mkdir -p ${packages_dist}
+
+for p in ${packages[@]}
+do
+  bazel build src/${p}:npm_package
+
+  # The output for "lib" should go into a directory named "material"
+  [[ ${p} = "lib" ]] && out_dir="material" || out_dir="${p}"
+
+  # Copy without preserving the read-only mode from bazel so that we can make final modifications
+  # to the generated package.
+  mkdir -p ${packages_dist}/${out_dir}
+  cp -r --no-preserve=mode ${bazel_bin}/src/${p}/npm_package/* ${packages_dist}/${out_dir}
+done
+
+# Update the root @angular/material metadata file to re-export metadata from each entry-point.
+./scripts/bazel/update-material-metadata-reexports.js
+
+# Update the Angular version used as a peerDependency for all of the packages.
+ng_version=$(echo "console.log(require('./build-config').angularVersion)" | node)
```

Comment:
Would it be better to use the `replacements` rule in `ng_package` for this?

See [here](https://github.com/angular/flex-layout/pull/652/files#diff-099cf06af08d6bbd3af78d00b6c5969aR29) and [here](https://github.com/angular/flex-layout/pull/652/files#diff-c64a394efa3dda4cf329bcad5192d1a7R1)

Reply:
+1 that's what it's for

Reply:
Done, didn't know that was there. I don't like that our gulp and bazel setups use a different constant, but it's  tolerable.

Reply:
The constant can be changed, that's just what `ngrx` used when they switched over.